whois-api/src/test/java/net/ripe/db/whois/api/httpserver/RewriteEngineTestIntegration.java

```diff
@@ -105,25 +105,13 @@ public void rest_lookup_with_rewrite() {
 
     @Test
     public void dont_allow_password_over_http() {
-        final ForbiddenException throwable = assertThrows(ForbiddenException.class, () ->
+        assertThrows(ForbiddenException.class, () ->
             RestTest.target(getPort(), "test/person/TP1-TEST?password=123")
                     .request()
                     .header(HttpHeaders.HOST, getHost(restApiBaseUrl))
                     .header(HttpHeader.X_FORWARDED_PROTO.toString(), HttpScheme.HTTP)
                     .get(WhoisResources.class)
         );
-        final String error = throwable.getResponse().readEntity(String.class);
```

Comment:
Can we still test response body?

Reply:
Response body is now empty, no HTML is returned by default. Need to manually test.